### ISSUE TITLE
Gather metadata to be added to the ro-crates

### ISSUE THREAD

**Issue by cpavloud** [emo-bon/observatory-bpns-crate#31]
-- **data owner**: EMBRC or the station or EMO BON?
According to [EMOBON’s Data Management Plan](https://www.embrc.eu/sites/default/files/publications/EMO_BON_Data_Management_Plan.pdf) “All Genomic Data and their metadata are owned by EMBRC being the entity responsible for their release. The Complementary (Meta)Data, Image Data and Occurrence Records collected are jointly owned by EMBRC and the operation stations”

 -- **organisation**: EMBRC but is there some ID for it?
Yes, EMBRC has [EDMO Id 5669](https://edmo.seadatanet.org/report/5669) 

 -- **data contact**:  EMO BON email address?

**Comment by cpavloud** [comment]
Sounds great!
FYI, there is a comment like this in all the observatory repositories. Maybe the repetition was unnecessary, but I wanted to be thorough.

**Comment by isanti** [comment]
noteworthy_env_cond check if this is added in all logsheets

**Comment by cpavloud** [comment]
Yes, probably it should refer to the observatory. I think it would be easier this way.

**Comment by isanti** [comment]
In the sediment log sheets add definitions for:
Tax_id
Samp_description
